client/src/views/components/PollsList/item.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import moment from 'moment';
+import { useDispatch } from 'react-redux';
+
+import { pollActions } from '../../../core/polls';
+
+const Item = ({ item }) => {
+  const dispatch = useDispatch();
+
+  return (
+    <li>
+      <button onClick={() => dispatch(pollActions.updatePollVote(item._id))}> Vote {item.votes} </button>
+      <h5>Topics Title</h5>
+      <p>
+        <span>By: Authors name</span>
+        <span>{moment(item.date_created).fromNow()}</span>
```

Comment:
`moment` is being deprecated, I think you can try using a more [lightweight alternative](https://bundlephobia.com/result?p=moment@2.29.1)

Reply:
So cool! It really helpful to improve my libs resource. Trying it...